[2021-06-25 15:18:52,264] {scheduler_job.py:182} INFO - Started process (PID=10178) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:18:52,265] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:18:52,266] {logging_mixin.py:104} INFO - [2021-06-25 15:18:52,266] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:18:52,279] {logging_mixin.py:104} INFO - [2021-06-25 15:18:52,276] {dagbag.py:320} ERROR - Failed to import: /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
Traceback (most recent call last):
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/airflow/models/dagbag.py", line 317, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py", line 4, in <module>
    from airflow.operators.docker_operator import DockerOperator
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/airflow/operators/docker_operator.py", line 23, in <module>
    from airflow.providers.docker.operators.docker import DockerOperator  # noqa
ModuleNotFoundError: No module named 'airflow.providers.docker'
[2021-06-25 15:18:52,280] {scheduler_job.py:645} WARNING - No viable dags retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:18:52,308] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.047 seconds
[2021-06-25 15:19:22,443] {scheduler_job.py:182} INFO - Started process (PID=10188) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:19:22,444] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:19:22,445] {logging_mixin.py:104} INFO - [2021-06-25 15:19:22,444] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:19:22,451] {logging_mixin.py:104} INFO - [2021-06-25 15:19:22,450] {dagbag.py:320} ERROR - Failed to import: /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
Traceback (most recent call last):
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/airflow/models/dagbag.py", line 317, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py", line 4, in <module>
    from airflow.operators.docker_operator import DockerOperator
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/airflow/operators/docker_operator.py", line 23, in <module>
    from airflow.providers.docker.operators.docker import DockerOperator  # noqa
ModuleNotFoundError: No module named 'airflow.providers.docker'
[2021-06-25 15:19:22,451] {scheduler_job.py:645} WARNING - No viable dags retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:19:22,478] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.038 seconds
[2021-06-25 15:19:52,643] {scheduler_job.py:182} INFO - Started process (PID=10203) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:19:52,644] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:19:52,645] {logging_mixin.py:104} INFO - [2021-06-25 15:19:52,644] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:19:52,651] {logging_mixin.py:104} INFO - [2021-06-25 15:19:52,651] {dagbag.py:320} ERROR - Failed to import: /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
Traceback (most recent call last):
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/airflow/models/dagbag.py", line 317, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py", line 4, in <module>
    from airflow.operators.docker_operator import DockerOperator
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/airflow/operators/docker_operator.py", line 23, in <module>
    from airflow.providers.docker.operators.docker import DockerOperator  # noqa
ModuleNotFoundError: No module named 'airflow.providers.docker'
[2021-06-25 15:19:52,652] {scheduler_job.py:645} WARNING - No viable dags retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:19:52,678] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.038 seconds
[2021-06-25 15:20:22,816] {scheduler_job.py:182} INFO - Started process (PID=10217) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:20:22,817] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:20:22,818] {logging_mixin.py:104} INFO - [2021-06-25 15:20:22,818] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:20:22,825] {logging_mixin.py:104} INFO - [2021-06-25 15:20:22,824] {dagbag.py:320} ERROR - Failed to import: /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
Traceback (most recent call last):
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/airflow/models/dagbag.py", line 317, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py", line 4, in <module>
    from airflow.operators.docker_operator import DockerOperator
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/airflow/operators/docker_operator.py", line 23, in <module>
    from airflow.providers.docker.operators.docker import DockerOperator  # noqa
ModuleNotFoundError: No module named 'airflow.providers.docker'
[2021-06-25 15:20:22,825] {scheduler_job.py:645} WARNING - No viable dags retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:20:22,851] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.037 seconds
[2021-06-25 15:20:53,001] {scheduler_job.py:182} INFO - Started process (PID=10227) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:20:53,003] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:20:53,004] {logging_mixin.py:104} INFO - [2021-06-25 15:20:53,004] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:20:53,010] {logging_mixin.py:104} INFO - [2021-06-25 15:20:53,009] {dagbag.py:320} ERROR - Failed to import: /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
Traceback (most recent call last):
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/airflow/models/dagbag.py", line 317, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py", line 4, in <module>
    from airflow.operators.docker_operator import DockerOperator
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/airflow/operators/docker_operator.py", line 23, in <module>
    from airflow.providers.docker.operators.docker import DockerOperator  # noqa
ModuleNotFoundError: No module named 'airflow.providers.docker'
[2021-06-25 15:20:53,011] {scheduler_job.py:645} WARNING - No viable dags retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:20:53,036] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.037 seconds
[2021-06-25 15:21:23,174] {scheduler_job.py:182} INFO - Started process (PID=10238) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:21:23,176] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:21:23,176] {logging_mixin.py:104} INFO - [2021-06-25 15:21:23,176] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:21:23,183] {logging_mixin.py:104} INFO - [2021-06-25 15:21:23,182] {dagbag.py:320} ERROR - Failed to import: /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
Traceback (most recent call last):
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/airflow/models/dagbag.py", line 317, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py", line 4, in <module>
    from airflow.operators.docker_operator import DockerOperator
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/airflow/operators/docker_operator.py", line 23, in <module>
    from airflow.providers.docker.operators.docker import DockerOperator  # noqa
ModuleNotFoundError: No module named 'airflow.providers.docker'
[2021-06-25 15:21:23,183] {scheduler_job.py:645} WARNING - No viable dags retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:21:23,210] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.039 seconds
[2021-06-25 15:21:53,364] {scheduler_job.py:182} INFO - Started process (PID=10247) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:21:53,364] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:21:53,365] {logging_mixin.py:104} INFO - [2021-06-25 15:21:53,365] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:21:53,372] {logging_mixin.py:104} INFO - [2021-06-25 15:21:53,371] {dagbag.py:320} ERROR - Failed to import: /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
Traceback (most recent call last):
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/airflow/models/dagbag.py", line 317, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py", line 4, in <module>
    from airflow.operators.docker_operator import DockerOperator
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/airflow/operators/docker_operator.py", line 23, in <module>
    from airflow.providers.docker.operators.docker import DockerOperator  # noqa
ModuleNotFoundError: No module named 'airflow.providers.docker'
[2021-06-25 15:21:53,372] {scheduler_job.py:645} WARNING - No viable dags retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:21:53,398] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.037 seconds
[2021-06-25 15:22:23,538] {scheduler_job.py:182} INFO - Started process (PID=10257) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:22:23,539] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:22:23,539] {logging_mixin.py:104} INFO - [2021-06-25 15:22:23,539] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:22:23,546] {logging_mixin.py:104} INFO - [2021-06-25 15:22:23,545] {dagbag.py:320} ERROR - Failed to import: /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
Traceback (most recent call last):
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/airflow/models/dagbag.py", line 317, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py", line 4, in <module>
    from airflow.operators.docker_operator import DockerOperator
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/airflow/operators/docker_operator.py", line 23, in <module>
    from airflow.providers.docker.operators.docker import DockerOperator  # noqa
ModuleNotFoundError: No module named 'airflow.providers.docker'
[2021-06-25 15:22:23,546] {scheduler_job.py:645} WARNING - No viable dags retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:22:23,573] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.038 seconds
[2021-06-25 15:22:53,741] {scheduler_job.py:182} INFO - Started process (PID=10267) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:22:53,742] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:22:53,742] {logging_mixin.py:104} INFO - [2021-06-25 15:22:53,742] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:22:53,749] {logging_mixin.py:104} INFO - [2021-06-25 15:22:53,748] {dagbag.py:320} ERROR - Failed to import: /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
Traceback (most recent call last):
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/airflow/models/dagbag.py", line 317, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py", line 4, in <module>
    from airflow.operators.docker_operator import DockerOperator
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/airflow/operators/docker_operator.py", line 23, in <module>
    from airflow.providers.docker.operators.docker import DockerOperator  # noqa
ModuleNotFoundError: No module named 'airflow.providers.docker'
[2021-06-25 15:22:53,749] {scheduler_job.py:645} WARNING - No viable dags retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:22:53,777] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.039 seconds
[2021-06-25 15:23:23,913] {scheduler_job.py:182} INFO - Started process (PID=10277) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:23:23,914] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:23:23,914] {logging_mixin.py:104} INFO - [2021-06-25 15:23:23,914] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:23:23,921] {logging_mixin.py:104} INFO - [2021-06-25 15:23:23,920] {dagbag.py:320} ERROR - Failed to import: /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
Traceback (most recent call last):
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/airflow/models/dagbag.py", line 317, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py", line 4, in <module>
    from airflow.operators.docker_operator import DockerOperator
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/airflow/operators/docker_operator.py", line 23, in <module>
    from airflow.providers.docker.operators.docker import DockerOperator  # noqa
ModuleNotFoundError: No module named 'airflow.providers.docker'
[2021-06-25 15:23:23,921] {scheduler_job.py:645} WARNING - No viable dags retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:23:23,950] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.040 seconds
[2021-06-25 15:23:54,120] {scheduler_job.py:182} INFO - Started process (PID=10287) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:23:54,120] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:23:54,121] {logging_mixin.py:104} INFO - [2021-06-25 15:23:54,121] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:23:54,128] {logging_mixin.py:104} INFO - [2021-06-25 15:23:54,128] {dagbag.py:320} ERROR - Failed to import: /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
Traceback (most recent call last):
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/airflow/models/dagbag.py", line 317, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py", line 4, in <module>
    from airflow.operators.docker_operator import DockerOperator
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/airflow/operators/docker_operator.py", line 23, in <module>
    from airflow.providers.docker.operators.docker import DockerOperator  # noqa
ModuleNotFoundError: No module named 'airflow.providers.docker'
[2021-06-25 15:23:54,129] {scheduler_job.py:645} WARNING - No viable dags retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:23:54,155] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.037 seconds
[2021-06-25 15:24:24,299] {scheduler_job.py:182} INFO - Started process (PID=10303) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:24:24,300] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:24:24,301] {logging_mixin.py:104} INFO - [2021-06-25 15:24:24,300] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:24:24,307] {logging_mixin.py:104} INFO - [2021-06-25 15:24:24,306] {dagbag.py:320} ERROR - Failed to import: /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
Traceback (most recent call last):
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/airflow/models/dagbag.py", line 317, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py", line 4, in <module>
    from airflow.operators.docker_operator import DockerOperator
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/airflow/operators/docker_operator.py", line 23, in <module>
    from airflow.providers.docker.operators.docker import DockerOperator  # noqa
ModuleNotFoundError: No module named 'airflow.providers.docker'
[2021-06-25 15:24:24,307] {scheduler_job.py:645} WARNING - No viable dags retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:24:24,332] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.036 seconds
[2021-06-25 15:24:54,498] {scheduler_job.py:182} INFO - Started process (PID=10312) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:24:54,499] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:24:54,499] {logging_mixin.py:104} INFO - [2021-06-25 15:24:54,499] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:24:54,506] {logging_mixin.py:104} INFO - [2021-06-25 15:24:54,505] {dagbag.py:320} ERROR - Failed to import: /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
Traceback (most recent call last):
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/airflow/models/dagbag.py", line 317, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py", line 4, in <module>
    from airflow.operators.docker_operator import DockerOperator
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/airflow/operators/docker_operator.py", line 23, in <module>
    from airflow.providers.docker.operators.docker import DockerOperator  # noqa
ModuleNotFoundError: No module named 'airflow.providers.docker'
[2021-06-25 15:24:54,507] {scheduler_job.py:645} WARNING - No viable dags retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:24:54,534] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.039 seconds
[2021-06-25 15:25:24,691] {scheduler_job.py:182} INFO - Started process (PID=10326) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:25:24,692] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:25:24,693] {logging_mixin.py:104} INFO - [2021-06-25 15:25:24,693] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:25:24,700] {logging_mixin.py:104} INFO - [2021-06-25 15:25:24,699] {dagbag.py:320} ERROR - Failed to import: /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
Traceback (most recent call last):
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/airflow/models/dagbag.py", line 317, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py", line 4, in <module>
    from airflow.operators.docker_operator import DockerOperator
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/airflow/operators/docker_operator.py", line 23, in <module>
    from airflow.providers.docker.operators.docker import DockerOperator  # noqa
ModuleNotFoundError: No module named 'airflow.providers.docker'
[2021-06-25 15:25:24,700] {scheduler_job.py:645} WARNING - No viable dags retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:25:24,725] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.036 seconds
[2021-06-25 15:25:54,900] {scheduler_job.py:182} INFO - Started process (PID=10347) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:25:54,901] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:25:54,902] {logging_mixin.py:104} INFO - [2021-06-25 15:25:54,901] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:25:54,908] {logging_mixin.py:104} INFO - [2021-06-25 15:25:54,908] {dagbag.py:320} ERROR - Failed to import: /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
Traceback (most recent call last):
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/airflow/models/dagbag.py", line 317, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py", line 4, in <module>
    from airflow.operators.docker_operator import DockerOperator
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/airflow/operators/docker_operator.py", line 23, in <module>
    from airflow.providers.docker.operators.docker import DockerOperator  # noqa
ModuleNotFoundError: No module named 'airflow.providers.docker'
[2021-06-25 15:25:54,909] {scheduler_job.py:645} WARNING - No viable dags retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:25:54,937] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.039 seconds
[2021-06-25 15:26:25,073] {scheduler_job.py:182} INFO - Started process (PID=10367) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:26:25,075] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:26:25,076] {logging_mixin.py:104} INFO - [2021-06-25 15:26:25,075] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:26:25,083] {logging_mixin.py:104} INFO - [2021-06-25 15:26:25,082] {dagbag.py:320} ERROR - Failed to import: /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
Traceback (most recent call last):
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/airflow/models/dagbag.py", line 317, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py", line 4, in <module>
    from airflow.operators.docker import DockerOperator
ModuleNotFoundError: No module named 'airflow.operators.docker'
[2021-06-25 15:26:25,083] {scheduler_job.py:645} WARNING - No viable dags retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:26:25,112] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.042 seconds
[2021-06-25 15:26:55,256] {scheduler_job.py:182} INFO - Started process (PID=10382) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:26:55,256] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:26:55,257] {logging_mixin.py:104} INFO - [2021-06-25 15:26:55,257] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:26:55,263] {logging_mixin.py:104} INFO - [2021-06-25 15:26:55,262] {dagbag.py:320} ERROR - Failed to import: /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
Traceback (most recent call last):
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/airflow/models/dagbag.py", line 317, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py", line 4, in <module>
    from airflow.operators.docker import DockerOperator
ModuleNotFoundError: No module named 'airflow.operators.docker'
[2021-06-25 15:26:55,263] {scheduler_job.py:645} WARNING - No viable dags retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:26:55,290] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.037 seconds
[2021-06-25 15:27:25,441] {scheduler_job.py:182} INFO - Started process (PID=10397) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:27:25,442] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:27:25,443] {logging_mixin.py:104} INFO - [2021-06-25 15:27:25,442] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:27:25,449] {logging_mixin.py:104} INFO - [2021-06-25 15:27:25,448] {dagbag.py:320} ERROR - Failed to import: /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
Traceback (most recent call last):
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/airflow/models/dagbag.py", line 317, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py", line 4, in <module>
    from airflow.operators.docker import DockerOperator
ModuleNotFoundError: No module named 'airflow.operators.docker'
[2021-06-25 15:27:25,449] {scheduler_job.py:645} WARNING - No viable dags retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:27:25,476] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.038 seconds
[2021-06-25 15:27:55,617] {scheduler_job.py:182} INFO - Started process (PID=10412) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:27:55,618] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:27:55,619] {logging_mixin.py:104} INFO - [2021-06-25 15:27:55,618] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:27:55,621] {logging_mixin.py:104} INFO - [2021-06-25 15:27:55,620] {dagbag.py:320} ERROR - Failed to import: /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
Traceback (most recent call last):
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/airflow/models/dagbag.py", line 317, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 674, in exec_module
  File "<frozen importlib._bootstrap_external>", line 781, in get_code
  File "<frozen importlib._bootstrap_external>", line 741, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py", line 5
    from airflow.providers.docker.
                                 ^
SyntaxError: invalid syntax
[2021-06-25 15:27:55,621] {scheduler_job.py:645} WARNING - No viable dags retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:27:55,647] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.033 seconds
[2021-06-25 15:28:25,808] {scheduler_job.py:182} INFO - Started process (PID=10432) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:28:25,809] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:28:25,809] {logging_mixin.py:104} INFO - [2021-06-25 15:28:25,809] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:28:25,816] {logging_mixin.py:104} INFO - [2021-06-25 15:28:25,816] {dagbag.py:320} ERROR - Failed to import: /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
Traceback (most recent call last):
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/airflow/models/dagbag.py", line 317, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py", line 4, in <module>
    from airflow.operators.docker_operator import DockerOperator
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/airflow/operators/docker_operator.py", line 23, in <module>
    from airflow.providers.docker.operators.docker import DockerOperator  # noqa
ModuleNotFoundError: No module named 'airflow.providers.docker'
[2021-06-25 15:28:25,817] {scheduler_job.py:645} WARNING - No viable dags retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:28:25,843] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.038 seconds
[2021-06-25 15:28:56,010] {scheduler_job.py:182} INFO - Started process (PID=10447) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:28:56,011] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:28:56,012] {logging_mixin.py:104} INFO - [2021-06-25 15:28:56,011] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:28:56,018] {logging_mixin.py:104} INFO - [2021-06-25 15:28:56,018] {dagbag.py:320} ERROR - Failed to import: /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
Traceback (most recent call last):
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/airflow/models/dagbag.py", line 317, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py", line 4, in <module>
    from airflow.operators.docker_operator import DockerOperator
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/airflow/operators/docker_operator.py", line 23, in <module>
    from airflow.providers.docker.operators.docker import DockerOperator  # noqa
ModuleNotFoundError: No module named 'airflow.providers.docker'
[2021-06-25 15:28:56,019] {scheduler_job.py:645} WARNING - No viable dags retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:28:56,048] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.040 seconds
[2021-06-25 15:29:26,187] {scheduler_job.py:182} INFO - Started process (PID=10456) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:29:26,188] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:29:26,189] {logging_mixin.py:104} INFO - [2021-06-25 15:29:26,189] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:29:26,195] {logging_mixin.py:104} INFO - [2021-06-25 15:29:26,194] {dagbag.py:320} ERROR - Failed to import: /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
Traceback (most recent call last):
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/airflow/models/dagbag.py", line 317, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py", line 4, in <module>
    from airflow.operators.docker_operator import DockerOperator
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/airflow/operators/docker_operator.py", line 23, in <module>
    from airflow.providers.docker.operators.docker import DockerOperator  # noqa
ModuleNotFoundError: No module named 'airflow.providers.docker'
[2021-06-25 15:29:26,195] {scheduler_job.py:645} WARNING - No viable dags retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:29:26,221] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.036 seconds
[2021-06-25 15:29:56,366] {scheduler_job.py:182} INFO - Started process (PID=10465) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:29:56,367] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:29:56,368] {logging_mixin.py:104} INFO - [2021-06-25 15:29:56,368] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:29:56,374] {logging_mixin.py:104} INFO - [2021-06-25 15:29:56,373] {dagbag.py:320} ERROR - Failed to import: /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
Traceback (most recent call last):
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/airflow/models/dagbag.py", line 317, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py", line 4, in <module>
    from airflow.operators.docker_operator import DockerOperator
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/airflow/operators/docker_operator.py", line 23, in <module>
    from airflow.providers.docker.operators.docker import DockerOperator  # noqa
ModuleNotFoundError: No module named 'airflow.providers.docker'
[2021-06-25 15:29:56,374] {scheduler_job.py:645} WARNING - No viable dags retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:29:56,402] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.039 seconds
[2021-06-25 15:30:26,533] {scheduler_job.py:182} INFO - Started process (PID=10497) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:30:26,533] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:30:26,534] {logging_mixin.py:104} INFO - [2021-06-25 15:30:26,534] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:30:26,541] {logging_mixin.py:104} INFO - [2021-06-25 15:30:26,540] {dagbag.py:320} ERROR - Failed to import: /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
Traceback (most recent call last):
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/airflow/models/dagbag.py", line 317, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py", line 4, in <module>
    from airflow.operators.docker_operator import DockerOperator
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/airflow/operators/docker_operator.py", line 23, in <module>
    from airflow.providers.docker.operators.docker import DockerOperator  # noqa
ModuleNotFoundError: No module named 'airflow.providers.docker'
[2021-06-25 15:30:26,541] {scheduler_job.py:645} WARNING - No viable dags retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:30:26,568] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.038 seconds
[2021-06-25 15:30:56,723] {scheduler_job.py:182} INFO - Started process (PID=10508) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:30:56,723] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:30:56,724] {logging_mixin.py:104} INFO - [2021-06-25 15:30:56,724] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:30:56,732] {logging_mixin.py:104} INFO - [2021-06-25 15:30:56,731] {dagbag.py:320} ERROR - Failed to import: /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
Traceback (most recent call last):
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/airflow/models/dagbag.py", line 317, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py", line 4, in <module>
    from airflow.operators.docker_operator import DockerOperator
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/airflow/operators/docker_operator.py", line 23, in <module>
    from airflow.providers.docker.operators.docker import DockerOperator  # noqa
ModuleNotFoundError: No module named 'airflow.providers.docker'
[2021-06-25 15:30:56,732] {scheduler_job.py:645} WARNING - No viable dags retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:30:56,759] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.039 seconds
[2021-06-25 15:31:26,891] {scheduler_job.py:182} INFO - Started process (PID=10594) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:31:26,892] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:31:26,893] {logging_mixin.py:104} INFO - [2021-06-25 15:31:26,892] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:31:26,976] {logging_mixin.py:104} INFO - [2021-06-25 15:31:26,975] {dagbag.py:320} ERROR - Failed to import: /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
Traceback (most recent call last):
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/airflow/models/dagbag.py", line 317, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py", line 5, in <module>
    from providers.docker.operators.docker import DockerOperator
ModuleNotFoundError: No module named 'providers'
[2021-06-25 15:31:26,976] {scheduler_job.py:645} WARNING - No viable dags retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:31:27,001] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.113 seconds
[2021-06-25 15:31:57,085] {scheduler_job.py:182} INFO - Started process (PID=10616) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:31:57,087] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:31:57,087] {logging_mixin.py:104} INFO - [2021-06-25 15:31:57,087] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:31:57,162] {logging_mixin.py:104} INFO - [2021-06-25 15:31:57,161] {dagbag.py:320} ERROR - Failed to import: /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
Traceback (most recent call last):
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/airflow/models/dagbag.py", line 317, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py", line 5, in <module>
    from providers.docker.operators.docker import DockerOperator
ModuleNotFoundError: No module named 'providers'
[2021-06-25 15:31:57,162] {scheduler_job.py:645} WARNING - No viable dags retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:31:57,188] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.105 seconds
[2021-06-25 15:32:27,340] {scheduler_job.py:182} INFO - Started process (PID=10627) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:32:27,341] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:32:27,342] {logging_mixin.py:104} INFO - [2021-06-25 15:32:27,342] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:32:27,425] {logging_mixin.py:104} INFO - [2021-06-25 15:32:27,422] {dagbag.py:393} ERROR - Failed to bag_dag: /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
Traceback (most recent call last):
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/croniter/croniter.py", line 156, in _alphaconv
    return cls.ALPHACONV[index][key]
KeyError: '/5'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/airflow/models/dagbag.py", line 388, in _process_modules
    croniter(dag.normalized_schedule_interval)
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/croniter/croniter.py", line 150, in __init__
    self.expanded, self.nth_weekday_of_month = self.expand(expr_format, hash_id=hash_id)
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/croniter/croniter.py", line 755, in expand
    return cls._expand(expr_format, hash_id=hash_id)
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/croniter/croniter.py", line 704, in _expand
    t = cls._alphaconv(i, t, expressions)
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/croniter/croniter.py", line 159, in _alphaconv
    "[{0}] is not acceptable".format(" ".join(expressions)))
croniter.croniter.CroniterNotAlphaError: [/5 * * * *] is not acceptable
[2021-06-25 15:32:27,425] {scheduler_job.py:645} WARNING - No viable dags retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:32:27,461] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.124 seconds
[2021-06-25 15:32:57,520] {scheduler_job.py:182} INFO - Started process (PID=10639) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:32:57,521] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:32:57,521] {logging_mixin.py:104} INFO - [2021-06-25 15:32:57,521] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:32:57,600] {logging_mixin.py:104} INFO - [2021-06-25 15:32:57,599] {dagbag.py:393} ERROR - Failed to bag_dag: /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
Traceback (most recent call last):
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/croniter/croniter.py", line 156, in _alphaconv
    return cls.ALPHACONV[index][key]
KeyError: '/5'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/airflow/models/dagbag.py", line 388, in _process_modules
    croniter(dag.normalized_schedule_interval)
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/croniter/croniter.py", line 150, in __init__
    self.expanded, self.nth_weekday_of_month = self.expand(expr_format, hash_id=hash_id)
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/croniter/croniter.py", line 755, in expand
    return cls._expand(expr_format, hash_id=hash_id)
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/croniter/croniter.py", line 704, in _expand
    t = cls._alphaconv(i, t, expressions)
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/croniter/croniter.py", line 159, in _alphaconv
    "[{0}] is not acceptable".format(" ".join(expressions)))
croniter.croniter.CroniterNotAlphaError: [/5 * * * *] is not acceptable
[2021-06-25 15:32:57,600] {scheduler_job.py:645} WARNING - No viable dags retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:32:57,626] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.109 seconds
[2021-06-25 15:33:27,697] {scheduler_job.py:182} INFO - Started process (PID=10648) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:33:27,698] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:33:27,698] {logging_mixin.py:104} INFO - [2021-06-25 15:33:27,698] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:33:27,774] {logging_mixin.py:104} INFO - [2021-06-25 15:33:27,773] {dagbag.py:393} ERROR - Failed to bag_dag: /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
Traceback (most recent call last):
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/croniter/croniter.py", line 156, in _alphaconv
    return cls.ALPHACONV[index][key]
KeyError: '/5'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/airflow/models/dagbag.py", line 388, in _process_modules
    croniter(dag.normalized_schedule_interval)
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/croniter/croniter.py", line 150, in __init__
    self.expanded, self.nth_weekday_of_month = self.expand(expr_format, hash_id=hash_id)
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/croniter/croniter.py", line 755, in expand
    return cls._expand(expr_format, hash_id=hash_id)
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/croniter/croniter.py", line 704, in _expand
    t = cls._alphaconv(i, t, expressions)
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/croniter/croniter.py", line 159, in _alphaconv
    "[{0}] is not acceptable".format(" ".join(expressions)))
croniter.croniter.CroniterNotAlphaError: [/5 * * * *] is not acceptable
[2021-06-25 15:33:27,775] {scheduler_job.py:645} WARNING - No viable dags retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:33:27,800] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.106 seconds
[2021-06-25 15:33:57,893] {scheduler_job.py:182} INFO - Started process (PID=10658) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:33:57,894] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:33:57,894] {logging_mixin.py:104} INFO - [2021-06-25 15:33:57,894] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:33:57,973] {logging_mixin.py:104} INFO - [2021-06-25 15:33:57,973] {dagbag.py:393} ERROR - Failed to bag_dag: /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
Traceback (most recent call last):
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/croniter/croniter.py", line 156, in _alphaconv
    return cls.ALPHACONV[index][key]
KeyError: '/5'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/airflow/models/dagbag.py", line 388, in _process_modules
    croniter(dag.normalized_schedule_interval)
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/croniter/croniter.py", line 150, in __init__
    self.expanded, self.nth_weekday_of_month = self.expand(expr_format, hash_id=hash_id)
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/croniter/croniter.py", line 755, in expand
    return cls._expand(expr_format, hash_id=hash_id)
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/croniter/croniter.py", line 704, in _expand
    t = cls._alphaconv(i, t, expressions)
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/croniter/croniter.py", line 159, in _alphaconv
    "[{0}] is not acceptable".format(" ".join(expressions)))
croniter.croniter.CroniterNotAlphaError: [/5 * * * *] is not acceptable
[2021-06-25 15:33:57,974] {scheduler_job.py:645} WARNING - No viable dags retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:33:57,999] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.109 seconds
[2021-06-25 15:34:28,065] {scheduler_job.py:182} INFO - Started process (PID=10675) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:34:28,066] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:34:28,067] {logging_mixin.py:104} INFO - [2021-06-25 15:34:28,066] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:34:28,143] {logging_mixin.py:104} INFO - [2021-06-25 15:34:28,142] {dagbag.py:393} ERROR - Failed to bag_dag: /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
Traceback (most recent call last):
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/croniter/croniter.py", line 156, in _alphaconv
    return cls.ALPHACONV[index][key]
KeyError: '/5'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/airflow/models/dagbag.py", line 388, in _process_modules
    croniter(dag.normalized_schedule_interval)
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/croniter/croniter.py", line 150, in __init__
    self.expanded, self.nth_weekday_of_month = self.expand(expr_format, hash_id=hash_id)
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/croniter/croniter.py", line 755, in expand
    return cls._expand(expr_format, hash_id=hash_id)
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/croniter/croniter.py", line 704, in _expand
    t = cls._alphaconv(i, t, expressions)
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/croniter/croniter.py", line 159, in _alphaconv
    "[{0}] is not acceptable".format(" ".join(expressions)))
croniter.croniter.CroniterNotAlphaError: [/5 * * * *] is not acceptable
[2021-06-25 15:34:28,143] {scheduler_job.py:645} WARNING - No viable dags retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:34:28,169] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.107 seconds
[2021-06-25 15:34:58,237] {scheduler_job.py:182} INFO - Started process (PID=10688) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:34:58,238] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:34:58,239] {logging_mixin.py:104} INFO - [2021-06-25 15:34:58,239] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:34:58,319] {logging_mixin.py:104} INFO - [2021-06-25 15:34:58,319] {dagbag.py:393} ERROR - Failed to bag_dag: /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
Traceback (most recent call last):
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/croniter/croniter.py", line 156, in _alphaconv
    return cls.ALPHACONV[index][key]
KeyError: '/5'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/airflow/models/dagbag.py", line 388, in _process_modules
    croniter(dag.normalized_schedule_interval)
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/croniter/croniter.py", line 150, in __init__
    self.expanded, self.nth_weekday_of_month = self.expand(expr_format, hash_id=hash_id)
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/croniter/croniter.py", line 755, in expand
    return cls._expand(expr_format, hash_id=hash_id)
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/croniter/croniter.py", line 704, in _expand
    t = cls._alphaconv(i, t, expressions)
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/croniter/croniter.py", line 159, in _alphaconv
    "[{0}] is not acceptable".format(" ".join(expressions)))
croniter.croniter.CroniterNotAlphaError: [/5 * * * *] is not acceptable
[2021-06-25 15:34:58,320] {scheduler_job.py:645} WARNING - No viable dags retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:34:58,346] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.111 seconds
[2021-06-25 15:35:28,410] {scheduler_job.py:182} INFO - Started process (PID=10702) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:35:28,411] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:35:28,412] {logging_mixin.py:104} INFO - [2021-06-25 15:35:28,412] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:35:28,489] {logging_mixin.py:104} INFO - [2021-06-25 15:35:28,488] {dagbag.py:393} ERROR - Failed to bag_dag: /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
Traceback (most recent call last):
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/croniter/croniter.py", line 156, in _alphaconv
    return cls.ALPHACONV[index][key]
KeyError: '/5'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/airflow/models/dagbag.py", line 388, in _process_modules
    croniter(dag.normalized_schedule_interval)
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/croniter/croniter.py", line 150, in __init__
    self.expanded, self.nth_weekday_of_month = self.expand(expr_format, hash_id=hash_id)
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/croniter/croniter.py", line 755, in expand
    return cls._expand(expr_format, hash_id=hash_id)
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/croniter/croniter.py", line 704, in _expand
    t = cls._alphaconv(i, t, expressions)
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/croniter/croniter.py", line 159, in _alphaconv
    "[{0}] is not acceptable".format(" ".join(expressions)))
croniter.croniter.CroniterNotAlphaError: [/5 * * * *] is not acceptable
[2021-06-25 15:35:28,490] {scheduler_job.py:645} WARNING - No viable dags retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:35:28,515] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.107 seconds
[2021-06-25 15:35:58,570] {scheduler_job.py:182} INFO - Started process (PID=10711) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:35:58,571] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:35:58,572] {logging_mixin.py:104} INFO - [2021-06-25 15:35:58,571] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:35:58,651] {logging_mixin.py:104} INFO - [2021-06-25 15:35:58,650] {dagbag.py:393} ERROR - Failed to bag_dag: /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
Traceback (most recent call last):
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/croniter/croniter.py", line 156, in _alphaconv
    return cls.ALPHACONV[index][key]
KeyError: '/5'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/airflow/models/dagbag.py", line 388, in _process_modules
    croniter(dag.normalized_schedule_interval)
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/croniter/croniter.py", line 150, in __init__
    self.expanded, self.nth_weekday_of_month = self.expand(expr_format, hash_id=hash_id)
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/croniter/croniter.py", line 755, in expand
    return cls._expand(expr_format, hash_id=hash_id)
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/croniter/croniter.py", line 704, in _expand
    t = cls._alphaconv(i, t, expressions)
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/croniter/croniter.py", line 159, in _alphaconv
    "[{0}] is not acceptable".format(" ".join(expressions)))
croniter.croniter.CroniterNotAlphaError: [/5 * * * *] is not acceptable
[2021-06-25 15:35:58,651] {scheduler_job.py:645} WARNING - No viable dags retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:35:58,679] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.111 seconds
[2021-06-25 15:36:28,738] {scheduler_job.py:182} INFO - Started process (PID=10722) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:36:28,739] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:36:28,739] {logging_mixin.py:104} INFO - [2021-06-25 15:36:28,739] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:36:28,818] {logging_mixin.py:104} INFO - [2021-06-25 15:36:28,817] {dagbag.py:393} ERROR - Failed to bag_dag: /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
Traceback (most recent call last):
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/croniter/croniter.py", line 156, in _alphaconv
    return cls.ALPHACONV[index][key]
KeyError: '/5'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/airflow/models/dagbag.py", line 388, in _process_modules
    croniter(dag.normalized_schedule_interval)
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/croniter/croniter.py", line 150, in __init__
    self.expanded, self.nth_weekday_of_month = self.expand(expr_format, hash_id=hash_id)
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/croniter/croniter.py", line 755, in expand
    return cls._expand(expr_format, hash_id=hash_id)
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/croniter/croniter.py", line 704, in _expand
    t = cls._alphaconv(i, t, expressions)
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/croniter/croniter.py", line 159, in _alphaconv
    "[{0}] is not acceptable".format(" ".join(expressions)))
croniter.croniter.CroniterNotAlphaError: [/5 * * * *] is not acceptable
[2021-06-25 15:36:28,818] {scheduler_job.py:645} WARNING - No viable dags retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:36:28,843] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.108 seconds
[2021-06-25 15:36:58,892] {scheduler_job.py:182} INFO - Started process (PID=10731) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:36:58,893] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:36:58,894] {logging_mixin.py:104} INFO - [2021-06-25 15:36:58,893] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:36:58,971] {logging_mixin.py:104} INFO - [2021-06-25 15:36:58,970] {dagbag.py:393} ERROR - Failed to bag_dag: /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
Traceback (most recent call last):
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/croniter/croniter.py", line 156, in _alphaconv
    return cls.ALPHACONV[index][key]
KeyError: '/5'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/airflow/models/dagbag.py", line 388, in _process_modules
    croniter(dag.normalized_schedule_interval)
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/croniter/croniter.py", line 150, in __init__
    self.expanded, self.nth_weekday_of_month = self.expand(expr_format, hash_id=hash_id)
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/croniter/croniter.py", line 755, in expand
    return cls._expand(expr_format, hash_id=hash_id)
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/croniter/croniter.py", line 704, in _expand
    t = cls._alphaconv(i, t, expressions)
  File "/Users/andre/Documents/docker_airflow/env/lib/python3.6/site-packages/croniter/croniter.py", line 159, in _alphaconv
    "[{0}] is not acceptable".format(" ".join(expressions)))
croniter.croniter.CroniterNotAlphaError: [/5 * * * *] is not acceptable
[2021-06-25 15:36:58,971] {scheduler_job.py:645} WARNING - No viable dags retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:36:58,996] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.107 seconds
[2021-06-25 15:37:29,065] {scheduler_job.py:182} INFO - Started process (PID=10750) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:37:29,067] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:37:29,068] {logging_mixin.py:104} INFO - [2021-06-25 15:37:29,067] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:37:29,148] {scheduler_job.py:643} INFO - DAG(s) dict_keys(['docker_dag']) retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:37:30,075] {logging_mixin.py:104} INFO - [2021-06-25 15:37:30,074] {dag.py:1832} INFO - Sync 1 DAGs
[2021-06-25 15:37:30,083] {logging_mixin.py:104} INFO - [2021-06-25 15:37:30,082] {dag.py:1851} INFO - Creating ORM DAG for docker_dag
[2021-06-25 15:37:30,093] {logging_mixin.py:104} INFO - [2021-06-25 15:37:30,093] {dag.py:2305} INFO - Setting next_dagrun for docker_dag to 2021-06-25T18:30:00+00:00
[2021-06-25 15:37:30,105] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 1.042 seconds
[2021-06-25 15:38:00,237] {scheduler_job.py:182} INFO - Started process (PID=10759) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:38:00,237] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:38:00,238] {logging_mixin.py:104} INFO - [2021-06-25 15:38:00,238] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:38:00,318] {scheduler_job.py:643} INFO - DAG(s) dict_keys(['docker_dag']) retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:38:00,347] {logging_mixin.py:104} INFO - [2021-06-25 15:38:00,346] {dag.py:1832} INFO - Sync 1 DAGs
[2021-06-25 15:38:00,366] {logging_mixin.py:104} INFO - [2021-06-25 15:38:00,365] {dag.py:2305} INFO - Setting next_dagrun for docker_dag to 2021-06-25T18:30:00+00:00
[2021-06-25 15:38:00,376] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.142 seconds
[2021-06-25 15:38:30,431] {scheduler_job.py:182} INFO - Started process (PID=10768) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:38:30,432] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:38:30,433] {logging_mixin.py:104} INFO - [2021-06-25 15:38:30,433] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:38:30,510] {scheduler_job.py:643} INFO - DAG(s) dict_keys(['docker_dag']) retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:38:30,540] {logging_mixin.py:104} INFO - [2021-06-25 15:38:30,540] {dag.py:1832} INFO - Sync 1 DAGs
[2021-06-25 15:38:30,560] {logging_mixin.py:104} INFO - [2021-06-25 15:38:30,560] {dag.py:2305} INFO - Setting next_dagrun for docker_dag to 2021-06-25T18:30:00+00:00
[2021-06-25 15:38:30,570] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.141 seconds
[2021-06-25 15:39:00,628] {scheduler_job.py:182} INFO - Started process (PID=10779) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:39:00,629] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:39:00,630] {logging_mixin.py:104} INFO - [2021-06-25 15:39:00,629] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:39:00,715] {scheduler_job.py:643} INFO - DAG(s) dict_keys(['docker_dag']) retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:39:00,744] {logging_mixin.py:104} INFO - [2021-06-25 15:39:00,743] {dag.py:1832} INFO - Sync 1 DAGs
[2021-06-25 15:39:00,763] {logging_mixin.py:104} INFO - [2021-06-25 15:39:00,762] {dag.py:2305} INFO - Setting next_dagrun for docker_dag to 2021-06-25T18:30:00+00:00
[2021-06-25 15:39:00,773] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.148 seconds
[2021-06-25 15:39:30,794] {scheduler_job.py:182} INFO - Started process (PID=10802) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:39:30,795] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:39:30,796] {logging_mixin.py:104} INFO - [2021-06-25 15:39:30,795] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:39:30,878] {scheduler_job.py:643} INFO - DAG(s) dict_keys(['docker_dag']) retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:39:30,905] {logging_mixin.py:104} INFO - [2021-06-25 15:39:30,905] {dag.py:1832} INFO - Sync 1 DAGs
[2021-06-25 15:39:30,922] {logging_mixin.py:104} INFO - [2021-06-25 15:39:30,922] {dag.py:2305} INFO - Setting next_dagrun for docker_dag to 2021-06-25T18:35:00+00:00
[2021-06-25 15:39:30,932] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.140 seconds
[2021-06-25 15:40:00,970] {scheduler_job.py:182} INFO - Started process (PID=10827) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:40:00,971] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:40:00,972] {logging_mixin.py:104} INFO - [2021-06-25 15:40:00,972] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:40:01,067] {scheduler_job.py:643} INFO - DAG(s) dict_keys(['docker_dag']) retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:40:01,315] {logging_mixin.py:104} INFO - [2021-06-25 15:40:01,315] {dag.py:1832} INFO - Sync 1 DAGs
[2021-06-25 15:40:01,329] {logging_mixin.py:104} INFO - [2021-06-25 15:40:01,329] {dag.py:2305} INFO - Setting next_dagrun for docker_dag to 2021-06-25T18:40:00+00:00
[2021-06-25 15:40:01,341] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.376 seconds
[2021-06-25 15:40:32,154] {scheduler_job.py:182} INFO - Started process (PID=10852) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:40:32,155] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:40:32,155] {logging_mixin.py:104} INFO - [2021-06-25 15:40:32,155] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:40:32,236] {scheduler_job.py:643} INFO - DAG(s) dict_keys(['docker_dag']) retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:40:32,264] {logging_mixin.py:104} INFO - [2021-06-25 15:40:32,264] {dag.py:1832} INFO - Sync 1 DAGs
[2021-06-25 15:40:32,281] {logging_mixin.py:104} INFO - [2021-06-25 15:40:32,281] {dag.py:2305} INFO - Setting next_dagrun for docker_dag to 2021-06-25T18:40:00+00:00
[2021-06-25 15:40:32,292] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.141 seconds
[2021-06-25 15:41:02,327] {scheduler_job.py:182} INFO - Started process (PID=10861) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:41:02,328] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:41:02,329] {logging_mixin.py:104} INFO - [2021-06-25 15:41:02,329] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:41:02,420] {scheduler_job.py:643} INFO - DAG(s) dict_keys(['docker_dag']) retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:41:02,448] {logging_mixin.py:104} INFO - [2021-06-25 15:41:02,448] {dag.py:1832} INFO - Sync 1 DAGs
[2021-06-25 15:41:02,464] {logging_mixin.py:104} INFO - [2021-06-25 15:41:02,464] {dag.py:2305} INFO - Setting next_dagrun for docker_dag to 2021-06-25T18:40:00+00:00
[2021-06-25 15:41:02,474] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.150 seconds
[2021-06-25 15:41:32,498] {scheduler_job.py:182} INFO - Started process (PID=10871) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:41:32,499] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:41:32,500] {logging_mixin.py:104} INFO - [2021-06-25 15:41:32,499] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:41:32,592] {scheduler_job.py:643} INFO - DAG(s) dict_keys(['docker_dag']) retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:41:32,618] {logging_mixin.py:104} INFO - [2021-06-25 15:41:32,618] {dag.py:1832} INFO - Sync 1 DAGs
[2021-06-25 15:41:32,635] {logging_mixin.py:104} INFO - [2021-06-25 15:41:32,634] {dag.py:2305} INFO - Setting next_dagrun for docker_dag to 2021-06-25T18:40:00+00:00
[2021-06-25 15:41:32,644] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.150 seconds
[2021-06-25 15:42:02,670] {scheduler_job.py:182} INFO - Started process (PID=10880) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:42:02,671] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:42:02,672] {logging_mixin.py:104} INFO - [2021-06-25 15:42:02,672] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:42:02,753] {scheduler_job.py:643} INFO - DAG(s) dict_keys(['docker_dag']) retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:42:02,782] {logging_mixin.py:104} INFO - [2021-06-25 15:42:02,782] {dag.py:1832} INFO - Sync 1 DAGs
[2021-06-25 15:42:02,802] {logging_mixin.py:104} INFO - [2021-06-25 15:42:02,801] {dag.py:2305} INFO - Setting next_dagrun for docker_dag to 2021-06-25T18:40:00+00:00
[2021-06-25 15:42:02,814] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.146 seconds
[2021-06-25 15:42:32,836] {scheduler_job.py:182} INFO - Started process (PID=10891) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:42:32,836] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:42:32,837] {logging_mixin.py:104} INFO - [2021-06-25 15:42:32,837] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:42:32,916] {scheduler_job.py:643} INFO - DAG(s) dict_keys(['docker_dag']) retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:42:32,946] {logging_mixin.py:104} INFO - [2021-06-25 15:42:32,945] {dag.py:1832} INFO - Sync 1 DAGs
[2021-06-25 15:42:32,966] {logging_mixin.py:104} INFO - [2021-06-25 15:42:32,966] {dag.py:2305} INFO - Setting next_dagrun for docker_dag to 2021-06-25T18:40:00+00:00
[2021-06-25 15:42:32,979] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.146 seconds
[2021-06-25 15:43:02,996] {scheduler_job.py:182} INFO - Started process (PID=10901) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:43:02,998] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:43:02,999] {logging_mixin.py:104} INFO - [2021-06-25 15:43:02,998] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:43:03,081] {scheduler_job.py:643} INFO - DAG(s) dict_keys(['docker_dag']) retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:43:03,108] {logging_mixin.py:104} INFO - [2021-06-25 15:43:03,107] {dag.py:1832} INFO - Sync 1 DAGs
[2021-06-25 15:43:03,124] {logging_mixin.py:104} INFO - [2021-06-25 15:43:03,124] {dag.py:2305} INFO - Setting next_dagrun for docker_dag to 2021-06-25T18:40:00+00:00
[2021-06-25 15:43:03,135] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.142 seconds
[2021-06-25 15:43:33,175] {scheduler_job.py:182} INFO - Started process (PID=10916) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:43:33,175] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:43:33,176] {logging_mixin.py:104} INFO - [2021-06-25 15:43:33,176] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:43:33,256] {scheduler_job.py:643} INFO - DAG(s) dict_keys(['docker_dag']) retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:43:33,284] {logging_mixin.py:104} INFO - [2021-06-25 15:43:33,284] {dag.py:1832} INFO - Sync 1 DAGs
[2021-06-25 15:43:33,301] {logging_mixin.py:104} INFO - [2021-06-25 15:43:33,301] {dag.py:2305} INFO - Setting next_dagrun for docker_dag to 2021-06-25T18:40:00+00:00
[2021-06-25 15:43:33,311] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.139 seconds
[2021-06-25 15:44:03,366] {scheduler_job.py:182} INFO - Started process (PID=10926) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:44:03,367] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:44:03,368] {logging_mixin.py:104} INFO - [2021-06-25 15:44:03,367] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:44:03,447] {scheduler_job.py:643} INFO - DAG(s) dict_keys(['docker_dag']) retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:44:03,475] {logging_mixin.py:104} INFO - [2021-06-25 15:44:03,475] {dag.py:1832} INFO - Sync 1 DAGs
[2021-06-25 15:44:03,492] {logging_mixin.py:104} INFO - [2021-06-25 15:44:03,491] {dag.py:2305} INFO - Setting next_dagrun for docker_dag to 2021-06-25T18:40:00+00:00
[2021-06-25 15:44:03,502] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.138 seconds
[2021-06-25 15:44:33,539] {scheduler_job.py:182} INFO - Started process (PID=10935) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:44:33,540] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:44:33,541] {logging_mixin.py:104} INFO - [2021-06-25 15:44:33,541] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:44:33,618] {scheduler_job.py:643} INFO - DAG(s) dict_keys(['docker_dag']) retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:44:33,648] {logging_mixin.py:104} INFO - [2021-06-25 15:44:33,647] {dag.py:1832} INFO - Sync 1 DAGs
[2021-06-25 15:44:33,664] {logging_mixin.py:104} INFO - [2021-06-25 15:44:33,664] {dag.py:2305} INFO - Setting next_dagrun for docker_dag to 2021-06-25T18:40:00+00:00
[2021-06-25 15:44:33,673] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.137 seconds
[2021-06-25 15:45:03,721] {scheduler_job.py:182} INFO - Started process (PID=10979) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:45:03,722] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:45:03,723] {logging_mixin.py:104} INFO - [2021-06-25 15:45:03,723] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:45:03,807] {scheduler_job.py:643} INFO - DAG(s) dict_keys(['docker_dag']) retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:45:03,836] {logging_mixin.py:104} INFO - [2021-06-25 15:45:03,836] {dag.py:1832} INFO - Sync 1 DAGs
[2021-06-25 15:45:03,854] {logging_mixin.py:104} INFO - [2021-06-25 15:45:03,854] {dag.py:2305} INFO - Setting next_dagrun for docker_dag to 2021-06-25T18:45:00+00:00
[2021-06-25 15:45:03,864] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.146 seconds
[2021-06-25 15:45:33,922] {scheduler_job.py:182} INFO - Started process (PID=10989) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:45:33,923] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:45:33,924] {logging_mixin.py:104} INFO - [2021-06-25 15:45:33,924] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:45:34,003] {scheduler_job.py:643} INFO - DAG(s) dict_keys(['docker_dag']) retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:45:34,034] {logging_mixin.py:104} INFO - [2021-06-25 15:45:34,034] {dag.py:1832} INFO - Sync 1 DAGs
[2021-06-25 15:45:34,051] {logging_mixin.py:104} INFO - [2021-06-25 15:45:34,051] {dag.py:2305} INFO - Setting next_dagrun for docker_dag to 2021-06-25T18:45:00+00:00
[2021-06-25 15:45:34,061] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.141 seconds
[2021-06-25 15:46:04,121] {scheduler_job.py:182} INFO - Started process (PID=10999) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:46:04,122] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:46:04,123] {logging_mixin.py:104} INFO - [2021-06-25 15:46:04,123] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:46:04,201] {scheduler_job.py:643} INFO - DAG(s) dict_keys(['docker_dag']) retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:46:04,230] {logging_mixin.py:104} INFO - [2021-06-25 15:46:04,230] {dag.py:1832} INFO - Sync 1 DAGs
[2021-06-25 15:46:04,250] {logging_mixin.py:104} INFO - [2021-06-25 15:46:04,250] {dag.py:2305} INFO - Setting next_dagrun for docker_dag to 2021-06-25T18:45:00+00:00
[2021-06-25 15:46:04,260] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.142 seconds
[2021-06-25 15:46:34,319] {scheduler_job.py:182} INFO - Started process (PID=11013) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:46:34,320] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:46:34,321] {logging_mixin.py:104} INFO - [2021-06-25 15:46:34,321] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:46:34,407] {scheduler_job.py:643} INFO - DAG(s) dict_keys(['docker_dag']) retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:46:34,435] {logging_mixin.py:104} INFO - [2021-06-25 15:46:34,435] {dag.py:1832} INFO - Sync 1 DAGs
[2021-06-25 15:46:34,453] {logging_mixin.py:104} INFO - [2021-06-25 15:46:34,452] {dag.py:2305} INFO - Setting next_dagrun for docker_dag to 2021-06-25T18:45:00+00:00
[2021-06-25 15:46:34,462] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.146 seconds
[2021-06-25 15:47:04,490] {scheduler_job.py:182} INFO - Started process (PID=11026) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:47:04,491] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:47:04,492] {logging_mixin.py:104} INFO - [2021-06-25 15:47:04,491] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:47:04,575] {scheduler_job.py:643} INFO - DAG(s) dict_keys(['docker_dag']) retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:47:04,601] {logging_mixin.py:104} INFO - [2021-06-25 15:47:04,601] {dag.py:1832} INFO - Sync 1 DAGs
[2021-06-25 15:47:04,618] {logging_mixin.py:104} INFO - [2021-06-25 15:47:04,618] {dag.py:2305} INFO - Setting next_dagrun for docker_dag to 2021-06-25T18:45:00+00:00
[2021-06-25 15:47:04,627] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.139 seconds
[2021-06-25 15:47:34,664] {scheduler_job.py:182} INFO - Started process (PID=11044) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:47:34,665] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:47:34,666] {logging_mixin.py:104} INFO - [2021-06-25 15:47:34,666] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:47:34,764] {scheduler_job.py:643} INFO - DAG(s) dict_keys(['docker_dag']) retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:47:34,793] {logging_mixin.py:104} INFO - [2021-06-25 15:47:34,792] {dag.py:1832} INFO - Sync 1 DAGs
[2021-06-25 15:47:34,810] {logging_mixin.py:104} INFO - [2021-06-25 15:47:34,810] {dag.py:2305} INFO - Setting next_dagrun for docker_dag to 2021-06-25T18:45:00+00:00
[2021-06-25 15:47:34,820] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.160 seconds
[2021-06-25 15:48:04,839] {scheduler_job.py:182} INFO - Started process (PID=11054) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:48:04,840] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:48:04,841] {logging_mixin.py:104} INFO - [2021-06-25 15:48:04,840] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:48:04,924] {scheduler_job.py:643} INFO - DAG(s) dict_keys(['docker_dag']) retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:48:04,951] {logging_mixin.py:104} INFO - [2021-06-25 15:48:04,951] {dag.py:1832} INFO - Sync 1 DAGs
[2021-06-25 15:48:04,969] {logging_mixin.py:104} INFO - [2021-06-25 15:48:04,969] {dag.py:2305} INFO - Setting next_dagrun for docker_dag to 2021-06-25T18:45:00+00:00
[2021-06-25 15:48:04,979] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.143 seconds
[2021-06-25 15:48:34,997] {scheduler_job.py:182} INFO - Started process (PID=11063) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:48:34,998] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:48:34,999] {logging_mixin.py:104} INFO - [2021-06-25 15:48:34,999] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:48:35,079] {scheduler_job.py:643} INFO - DAG(s) dict_keys(['docker_dag']) retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:48:35,107] {logging_mixin.py:104} INFO - [2021-06-25 15:48:35,107] {dag.py:1832} INFO - Sync 1 DAGs
[2021-06-25 15:48:35,124] {logging_mixin.py:104} INFO - [2021-06-25 15:48:35,124] {dag.py:2305} INFO - Setting next_dagrun for docker_dag to 2021-06-25T18:45:00+00:00
[2021-06-25 15:48:35,134] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.140 seconds
[2021-06-25 15:49:05,181] {scheduler_job.py:182} INFO - Started process (PID=11073) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:49:05,182] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:49:05,183] {logging_mixin.py:104} INFO - [2021-06-25 15:49:05,183] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:49:05,261] {scheduler_job.py:643} INFO - DAG(s) dict_keys(['docker_dag']) retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:49:05,287] {logging_mixin.py:104} INFO - [2021-06-25 15:49:05,287] {dag.py:1832} INFO - Sync 1 DAGs
[2021-06-25 15:49:05,304] {logging_mixin.py:104} INFO - [2021-06-25 15:49:05,304] {dag.py:2305} INFO - Setting next_dagrun for docker_dag to 2021-06-25T18:45:00+00:00
[2021-06-25 15:49:05,314] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.135 seconds
[2021-06-25 15:49:35,349] {scheduler_job.py:182} INFO - Started process (PID=11083) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:49:35,350] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:49:35,351] {logging_mixin.py:104} INFO - [2021-06-25 15:49:35,350] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:49:35,429] {scheduler_job.py:643} INFO - DAG(s) dict_keys(['docker_dag']) retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:49:35,458] {logging_mixin.py:104} INFO - [2021-06-25 15:49:35,457] {dag.py:1832} INFO - Sync 1 DAGs
[2021-06-25 15:49:35,474] {logging_mixin.py:104} INFO - [2021-06-25 15:49:35,474] {dag.py:2305} INFO - Setting next_dagrun for docker_dag to 2021-06-25T18:45:00+00:00
[2021-06-25 15:49:35,484] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.138 seconds
[2021-06-25 15:50:05,515] {scheduler_job.py:182} INFO - Started process (PID=11113) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:50:05,516] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:50:05,516] {logging_mixin.py:104} INFO - [2021-06-25 15:50:05,516] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:50:05,595] {scheduler_job.py:643} INFO - DAG(s) dict_keys(['docker_dag']) retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:50:05,623] {logging_mixin.py:104} INFO - [2021-06-25 15:50:05,623] {dag.py:1832} INFO - Sync 1 DAGs
[2021-06-25 15:50:05,640] {logging_mixin.py:104} INFO - [2021-06-25 15:50:05,640] {dag.py:2305} INFO - Setting next_dagrun for docker_dag to 2021-06-25T18:50:00+00:00
[2021-06-25 15:50:05,650] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.138 seconds
[2021-06-25 15:50:35,690] {scheduler_job.py:182} INFO - Started process (PID=11126) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:50:35,691] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:50:35,691] {logging_mixin.py:104} INFO - [2021-06-25 15:50:35,691] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:50:35,771] {scheduler_job.py:643} INFO - DAG(s) dict_keys(['docker_dag']) retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:50:35,799] {logging_mixin.py:104} INFO - [2021-06-25 15:50:35,799] {dag.py:1832} INFO - Sync 1 DAGs
[2021-06-25 15:50:35,815] {logging_mixin.py:104} INFO - [2021-06-25 15:50:35,815] {dag.py:2305} INFO - Setting next_dagrun for docker_dag to 2021-06-25T18:50:00+00:00
[2021-06-25 15:50:35,825] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.138 seconds
[2021-06-25 15:51:05,859] {scheduler_job.py:182} INFO - Started process (PID=11141) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:51:05,860] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:51:05,861] {logging_mixin.py:104} INFO - [2021-06-25 15:51:05,860] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:51:05,940] {scheduler_job.py:643} INFO - DAG(s) dict_keys(['docker_dag']) retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:51:05,969] {logging_mixin.py:104} INFO - [2021-06-25 15:51:05,969] {dag.py:1832} INFO - Sync 1 DAGs
[2021-06-25 15:51:05,988] {logging_mixin.py:104} INFO - [2021-06-25 15:51:05,988] {dag.py:2305} INFO - Setting next_dagrun for docker_dag to 2021-06-25T18:50:00+00:00
[2021-06-25 15:51:06,002] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.146 seconds
[2021-06-25 15:51:36,024] {scheduler_job.py:182} INFO - Started process (PID=11156) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:51:36,024] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:51:36,025] {logging_mixin.py:104} INFO - [2021-06-25 15:51:36,025] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:51:36,109] {scheduler_job.py:643} INFO - DAG(s) dict_keys(['docker_dag']) retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:51:36,135] {logging_mixin.py:104} INFO - [2021-06-25 15:51:36,135] {dag.py:1832} INFO - Sync 1 DAGs
[2021-06-25 15:51:36,152] {logging_mixin.py:104} INFO - [2021-06-25 15:51:36,151] {dag.py:2305} INFO - Setting next_dagrun for docker_dag to 2021-06-25T18:50:00+00:00
[2021-06-25 15:51:36,162] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.141 seconds
[2021-06-25 15:52:06,191] {scheduler_job.py:182} INFO - Started process (PID=11169) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:52:06,192] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:52:06,193] {logging_mixin.py:104} INFO - [2021-06-25 15:52:06,193] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:52:06,275] {scheduler_job.py:643} INFO - DAG(s) dict_keys(['docker_dag']) retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:52:06,304] {logging_mixin.py:104} INFO - [2021-06-25 15:52:06,303] {dag.py:1832} INFO - Sync 1 DAGs
[2021-06-25 15:52:06,321] {logging_mixin.py:104} INFO - [2021-06-25 15:52:06,321] {dag.py:2305} INFO - Setting next_dagrun for docker_dag to 2021-06-25T18:50:00+00:00
[2021-06-25 15:52:06,338] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.149 seconds
[2021-06-25 15:52:36,372] {scheduler_job.py:182} INFO - Started process (PID=11187) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:52:36,373] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:52:36,373] {logging_mixin.py:104} INFO - [2021-06-25 15:52:36,373] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:52:36,451] {scheduler_job.py:643} INFO - DAG(s) dict_keys(['docker_dag']) retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:52:36,478] {logging_mixin.py:104} INFO - [2021-06-25 15:52:36,478] {dag.py:1832} INFO - Sync 1 DAGs
[2021-06-25 15:52:36,494] {logging_mixin.py:104} INFO - [2021-06-25 15:52:36,494] {dag.py:2305} INFO - Setting next_dagrun for docker_dag to 2021-06-25T18:50:00+00:00
[2021-06-25 15:52:36,504] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.135 seconds
[2021-06-25 15:53:06,535] {scheduler_job.py:182} INFO - Started process (PID=11196) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:53:06,536] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:53:06,537] {logging_mixin.py:104} INFO - [2021-06-25 15:53:06,537] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:53:06,618] {scheduler_job.py:643} INFO - DAG(s) dict_keys(['docker_dag']) retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:53:06,645] {logging_mixin.py:104} INFO - [2021-06-25 15:53:06,645] {dag.py:1832} INFO - Sync 1 DAGs
[2021-06-25 15:53:06,662] {logging_mixin.py:104} INFO - [2021-06-25 15:53:06,661] {dag.py:2305} INFO - Setting next_dagrun for docker_dag to 2021-06-25T18:50:00+00:00
[2021-06-25 15:53:06,671] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.139 seconds
[2021-06-25 15:53:36,714] {scheduler_job.py:182} INFO - Started process (PID=11208) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:53:36,714] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:53:36,715] {logging_mixin.py:104} INFO - [2021-06-25 15:53:36,715] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:53:36,792] {scheduler_job.py:643} INFO - DAG(s) dict_keys(['docker_dag']) retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:53:36,821] {logging_mixin.py:104} INFO - [2021-06-25 15:53:36,821] {dag.py:1832} INFO - Sync 1 DAGs
[2021-06-25 15:53:36,840] {logging_mixin.py:104} INFO - [2021-06-25 15:53:36,840] {dag.py:2305} INFO - Setting next_dagrun for docker_dag to 2021-06-25T18:50:00+00:00
[2021-06-25 15:53:36,851] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.140 seconds
[2021-06-25 15:54:06,909] {scheduler_job.py:182} INFO - Started process (PID=11217) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:54:06,910] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:54:06,911] {logging_mixin.py:104} INFO - [2021-06-25 15:54:06,910] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:54:06,988] {scheduler_job.py:643} INFO - DAG(s) dict_keys(['docker_dag']) retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:54:07,019] {logging_mixin.py:104} INFO - [2021-06-25 15:54:07,019] {dag.py:1832} INFO - Sync 1 DAGs
[2021-06-25 15:54:07,037] {logging_mixin.py:104} INFO - [2021-06-25 15:54:07,037] {dag.py:2305} INFO - Setting next_dagrun for docker_dag to 2021-06-25T18:50:00+00:00
[2021-06-25 15:54:07,048] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.142 seconds
[2021-06-25 15:54:37,087] {scheduler_job.py:182} INFO - Started process (PID=11272) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:54:37,088] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:54:37,089] {logging_mixin.py:104} INFO - [2021-06-25 15:54:37,089] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:54:37,172] {scheduler_job.py:643} INFO - DAG(s) dict_keys(['docker_dag']) retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:54:37,200] {logging_mixin.py:104} INFO - [2021-06-25 15:54:37,200] {dag.py:1832} INFO - Sync 1 DAGs
[2021-06-25 15:54:37,218] {logging_mixin.py:104} INFO - [2021-06-25 15:54:37,218] {dag.py:2305} INFO - Setting next_dagrun for docker_dag to 2021-06-25T18:50:00+00:00
[2021-06-25 15:54:37,229] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.144 seconds
[2021-06-25 15:55:07,249] {scheduler_job.py:182} INFO - Started process (PID=11311) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:55:07,249] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:55:07,250] {logging_mixin.py:104} INFO - [2021-06-25 15:55:07,250] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:55:07,331] {scheduler_job.py:643} INFO - DAG(s) dict_keys(['docker_dag']) retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:55:07,358] {logging_mixin.py:104} INFO - [2021-06-25 15:55:07,358] {dag.py:1832} INFO - Sync 1 DAGs
[2021-06-25 15:55:07,375] {logging_mixin.py:104} INFO - [2021-06-25 15:55:07,375] {dag.py:2305} INFO - Setting next_dagrun for docker_dag to 2021-06-25T18:55:00+00:00
[2021-06-25 15:55:07,386] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.140 seconds
[2021-06-25 15:55:37,414] {scheduler_job.py:182} INFO - Started process (PID=11342) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:55:37,415] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:55:37,416] {logging_mixin.py:104} INFO - [2021-06-25 15:55:37,415] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:55:37,500] {scheduler_job.py:643} INFO - DAG(s) dict_keys(['docker_dag']) retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:55:37,527] {logging_mixin.py:104} INFO - [2021-06-25 15:55:37,527] {dag.py:1832} INFO - Sync 1 DAGs
[2021-06-25 15:55:37,544] {logging_mixin.py:104} INFO - [2021-06-25 15:55:37,544] {dag.py:2305} INFO - Setting next_dagrun for docker_dag to 2021-06-25T18:55:00+00:00
[2021-06-25 15:55:37,555] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.143 seconds
[2021-06-25 15:56:07,587] {scheduler_job.py:182} INFO - Started process (PID=11351) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:56:07,588] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:56:07,589] {logging_mixin.py:104} INFO - [2021-06-25 15:56:07,589] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:56:07,675] {scheduler_job.py:643} INFO - DAG(s) dict_keys(['docker_dag']) retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:56:07,703] {logging_mixin.py:104} INFO - [2021-06-25 15:56:07,703] {dag.py:1832} INFO - Sync 1 DAGs
[2021-06-25 15:56:07,720] {logging_mixin.py:104} INFO - [2021-06-25 15:56:07,720] {dag.py:2305} INFO - Setting next_dagrun for docker_dag to 2021-06-25T18:55:00+00:00
[2021-06-25 15:56:07,730] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.145 seconds
[2021-06-25 15:56:37,758] {scheduler_job.py:182} INFO - Started process (PID=11361) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:56:37,758] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:56:37,759] {logging_mixin.py:104} INFO - [2021-06-25 15:56:37,759] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:56:37,839] {scheduler_job.py:643} INFO - DAG(s) dict_keys(['docker_dag']) retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:56:37,867] {logging_mixin.py:104} INFO - [2021-06-25 15:56:37,867] {dag.py:1832} INFO - Sync 1 DAGs
[2021-06-25 15:56:37,884] {logging_mixin.py:104} INFO - [2021-06-25 15:56:37,884] {dag.py:2305} INFO - Setting next_dagrun for docker_dag to 2021-06-25T18:55:00+00:00
[2021-06-25 15:56:37,894] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.139 seconds
[2021-06-25 15:57:07,931] {scheduler_job.py:182} INFO - Started process (PID=11376) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:57:07,932] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:57:07,933] {logging_mixin.py:104} INFO - [2021-06-25 15:57:07,932] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:57:08,028] {scheduler_job.py:643} INFO - DAG(s) dict_keys(['docker_dag']) retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:57:08,060] {logging_mixin.py:104} INFO - [2021-06-25 15:57:08,060] {dag.py:1832} INFO - Sync 1 DAGs
[2021-06-25 15:57:08,079] {logging_mixin.py:104} INFO - [2021-06-25 15:57:08,079] {dag.py:2305} INFO - Setting next_dagrun for docker_dag to 2021-06-25T18:55:00+00:00
[2021-06-25 15:57:08,091] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.162 seconds
[2021-06-25 15:57:39,093] {scheduler_job.py:182} INFO - Started process (PID=11388) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:57:39,094] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:57:39,095] {logging_mixin.py:104} INFO - [2021-06-25 15:57:39,094] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:57:39,180] {scheduler_job.py:643} INFO - DAG(s) dict_keys(['docker_dag']) retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:57:39,212] {logging_mixin.py:104} INFO - [2021-06-25 15:57:39,212] {dag.py:1832} INFO - Sync 1 DAGs
[2021-06-25 15:57:39,233] {logging_mixin.py:104} INFO - [2021-06-25 15:57:39,232] {dag.py:2305} INFO - Setting next_dagrun for docker_dag to 2021-06-25T18:55:00+00:00
[2021-06-25 15:57:39,247] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.157 seconds
[2021-06-25 15:58:10,266] {scheduler_job.py:182} INFO - Started process (PID=11397) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:58:10,267] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:58:10,267] {logging_mixin.py:104} INFO - [2021-06-25 15:58:10,267] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:58:10,349] {scheduler_job.py:643} INFO - DAG(s) dict_keys(['docker_dag']) retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:58:10,374] {logging_mixin.py:104} INFO - [2021-06-25 15:58:10,374] {dag.py:1832} INFO - Sync 1 DAGs
[2021-06-25 15:58:10,390] {logging_mixin.py:104} INFO - [2021-06-25 15:58:10,390] {dag.py:2305} INFO - Setting next_dagrun for docker_dag to 2021-06-25T18:55:00+00:00
[2021-06-25 15:58:10,400] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.137 seconds
[2021-06-25 15:58:40,429] {scheduler_job.py:182} INFO - Started process (PID=11422) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:58:40,430] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:58:40,431] {logging_mixin.py:104} INFO - [2021-06-25 15:58:40,430] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:58:40,525] {scheduler_job.py:643} INFO - DAG(s) dict_keys(['docker_dag']) retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:58:40,551] {logging_mixin.py:104} INFO - [2021-06-25 15:58:40,551] {dag.py:1832} INFO - Sync 1 DAGs
[2021-06-25 15:58:40,568] {logging_mixin.py:104} INFO - [2021-06-25 15:58:40,568] {dag.py:2305} INFO - Setting next_dagrun for docker_dag to 2021-06-25T18:55:00+00:00
[2021-06-25 15:58:40,578] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.153 seconds
[2021-06-25 15:59:10,631] {scheduler_job.py:182} INFO - Started process (PID=11437) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:59:10,632] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:59:10,634] {logging_mixin.py:104} INFO - [2021-06-25 15:59:10,633] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:59:10,721] {scheduler_job.py:643} INFO - DAG(s) dict_keys(['docker_dag']) retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:59:10,748] {logging_mixin.py:104} INFO - [2021-06-25 15:59:10,748] {dag.py:1832} INFO - Sync 1 DAGs
[2021-06-25 15:59:10,765] {logging_mixin.py:104} INFO - [2021-06-25 15:59:10,764] {dag.py:2305} INFO - Setting next_dagrun for docker_dag to 2021-06-25T18:55:00+00:00
[2021-06-25 15:59:10,775] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.147 seconds
[2021-06-25 15:59:40,803] {scheduler_job.py:182} INFO - Started process (PID=11454) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:59:40,804] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 15:59:40,804] {logging_mixin.py:104} INFO - [2021-06-25 15:59:40,804] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:59:40,898] {scheduler_job.py:643} INFO - DAG(s) dict_keys(['docker_dag']) retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 15:59:40,926] {logging_mixin.py:104} INFO - [2021-06-25 15:59:40,926] {dag.py:1832} INFO - Sync 1 DAGs
[2021-06-25 15:59:40,943] {logging_mixin.py:104} INFO - [2021-06-25 15:59:40,943] {dag.py:2305} INFO - Setting next_dagrun for docker_dag to 2021-06-25T18:55:00+00:00
[2021-06-25 15:59:40,953] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.154 seconds
[2021-06-25 16:00:10,977] {scheduler_job.py:182} INFO - Started process (PID=11505) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 16:00:10,977] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 16:00:10,978] {logging_mixin.py:104} INFO - [2021-06-25 16:00:10,978] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 16:00:11,061] {scheduler_job.py:643} INFO - DAG(s) dict_keys(['docker_dag']) retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 16:00:11,089] {logging_mixin.py:104} INFO - [2021-06-25 16:00:11,088] {dag.py:1832} INFO - Sync 1 DAGs
[2021-06-25 16:00:11,113] {logging_mixin.py:104} INFO - [2021-06-25 16:00:11,113] {dag.py:2305} INFO - Setting next_dagrun for docker_dag to 2021-06-25T19:00:00+00:00
[2021-06-25 16:00:11,124] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.150 seconds
[2021-06-25 16:00:41,146] {scheduler_job.py:182} INFO - Started process (PID=11515) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 16:00:41,147] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 16:00:41,148] {logging_mixin.py:104} INFO - [2021-06-25 16:00:41,147] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 16:00:41,228] {scheduler_job.py:643} INFO - DAG(s) dict_keys(['docker_dag']) retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 16:00:41,257] {logging_mixin.py:104} INFO - [2021-06-25 16:00:41,257] {dag.py:1832} INFO - Sync 1 DAGs
[2021-06-25 16:00:41,274] {logging_mixin.py:104} INFO - [2021-06-25 16:00:41,274] {dag.py:2305} INFO - Setting next_dagrun for docker_dag to 2021-06-25T19:00:00+00:00
[2021-06-25 16:00:41,284] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.141 seconds
[2021-06-25 16:01:11,319] {scheduler_job.py:182} INFO - Started process (PID=11524) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 16:01:11,320] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 16:01:11,321] {logging_mixin.py:104} INFO - [2021-06-25 16:01:11,321] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 16:01:11,401] {scheduler_job.py:643} INFO - DAG(s) dict_keys(['docker_dag']) retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 16:01:11,427] {logging_mixin.py:104} INFO - [2021-06-25 16:01:11,426] {dag.py:1832} INFO - Sync 1 DAGs
[2021-06-25 16:01:11,443] {logging_mixin.py:104} INFO - [2021-06-25 16:01:11,443] {dag.py:2305} INFO - Setting next_dagrun for docker_dag to 2021-06-25T19:00:00+00:00
[2021-06-25 16:01:11,452] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.135 seconds
[2021-06-25 16:01:41,471] {scheduler_job.py:182} INFO - Started process (PID=11540) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 16:01:41,472] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 16:01:41,473] {logging_mixin.py:104} INFO - [2021-06-25 16:01:41,472] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 16:01:41,553] {scheduler_job.py:643} INFO - DAG(s) dict_keys(['docker_dag']) retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 16:01:41,582] {logging_mixin.py:104} INFO - [2021-06-25 16:01:41,581] {dag.py:1832} INFO - Sync 1 DAGs
[2021-06-25 16:01:41,598] {logging_mixin.py:104} INFO - [2021-06-25 16:01:41,598] {dag.py:2305} INFO - Setting next_dagrun for docker_dag to 2021-06-25T19:00:00+00:00
[2021-06-25 16:01:41,609] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.140 seconds
[2021-06-25 16:02:11,648] {scheduler_job.py:182} INFO - Started process (PID=11556) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 16:02:11,649] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 16:02:11,650] {logging_mixin.py:104} INFO - [2021-06-25 16:02:11,650] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 16:02:11,733] {scheduler_job.py:643} INFO - DAG(s) dict_keys(['docker_dag']) retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 16:02:11,761] {logging_mixin.py:104} INFO - [2021-06-25 16:02:11,761] {dag.py:1832} INFO - Sync 1 DAGs
[2021-06-25 16:02:11,779] {logging_mixin.py:104} INFO - [2021-06-25 16:02:11,779] {dag.py:2305} INFO - Setting next_dagrun for docker_dag to 2021-06-25T19:00:00+00:00
[2021-06-25 16:02:11,789] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.145 seconds
[2021-06-25 16:02:42,734] {scheduler_job.py:182} INFO - Started process (PID=11569) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 16:02:42,735] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 16:02:42,736] {logging_mixin.py:104} INFO - [2021-06-25 16:02:42,736] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 16:02:42,816] {scheduler_job.py:643} INFO - DAG(s) dict_keys(['docker_dag']) retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 16:02:42,845] {logging_mixin.py:104} INFO - [2021-06-25 16:02:42,844] {dag.py:1832} INFO - Sync 1 DAGs
[2021-06-25 16:02:42,861] {logging_mixin.py:104} INFO - [2021-06-25 16:02:42,861] {dag.py:2305} INFO - Setting next_dagrun for docker_dag to 2021-06-25T19:00:00+00:00
[2021-06-25 16:02:42,871] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.140 seconds
[2021-06-25 16:03:12,972] {scheduler_job.py:182} INFO - Started process (PID=11578) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 16:03:12,972] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 16:03:12,973] {logging_mixin.py:104} INFO - [2021-06-25 16:03:12,973] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 16:03:13,055] {scheduler_job.py:643} INFO - DAG(s) dict_keys(['docker_dag']) retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 16:03:13,082] {logging_mixin.py:104} INFO - [2021-06-25 16:03:13,082] {dag.py:1832} INFO - Sync 1 DAGs
[2021-06-25 16:03:13,099] {logging_mixin.py:104} INFO - [2021-06-25 16:03:13,099] {dag.py:2305} INFO - Setting next_dagrun for docker_dag to 2021-06-25T19:00:00+00:00
[2021-06-25 16:03:13,109] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.140 seconds
[2021-06-25 16:03:43,196] {scheduler_job.py:182} INFO - Started process (PID=11590) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 16:03:43,196] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 16:03:43,197] {logging_mixin.py:104} INFO - [2021-06-25 16:03:43,197] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 16:03:43,275] {scheduler_job.py:643} INFO - DAG(s) dict_keys(['docker_dag']) retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 16:03:43,305] {logging_mixin.py:104} INFO - [2021-06-25 16:03:43,304] {dag.py:1832} INFO - Sync 1 DAGs
[2021-06-25 16:03:43,323] {logging_mixin.py:104} INFO - [2021-06-25 16:03:43,322] {dag.py:2305} INFO - Setting next_dagrun for docker_dag to 2021-06-25T19:00:00+00:00
[2021-06-25 16:03:43,335] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.142 seconds
[2021-06-25 16:04:13,385] {scheduler_job.py:182} INFO - Started process (PID=11599) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 16:04:13,386] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 16:04:13,387] {logging_mixin.py:104} INFO - [2021-06-25 16:04:13,387] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 16:04:13,468] {scheduler_job.py:643} INFO - DAG(s) dict_keys(['docker_dag']) retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 16:04:13,496] {logging_mixin.py:104} INFO - [2021-06-25 16:04:13,496] {dag.py:1832} INFO - Sync 1 DAGs
[2021-06-25 16:04:13,513] {logging_mixin.py:104} INFO - [2021-06-25 16:04:13,513] {dag.py:2305} INFO - Setting next_dagrun for docker_dag to 2021-06-25T19:00:00+00:00
[2021-06-25 16:04:13,524] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.142 seconds
[2021-06-25 16:04:43,555] {scheduler_job.py:182} INFO - Started process (PID=11609) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 16:04:43,556] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 16:04:43,557] {logging_mixin.py:104} INFO - [2021-06-25 16:04:43,556] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 16:04:43,635] {scheduler_job.py:643} INFO - DAG(s) dict_keys(['docker_dag']) retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 16:04:43,661] {logging_mixin.py:104} INFO - [2021-06-25 16:04:43,661] {dag.py:1832} INFO - Sync 1 DAGs
[2021-06-25 16:04:43,677] {logging_mixin.py:104} INFO - [2021-06-25 16:04:43,677] {dag.py:2305} INFO - Setting next_dagrun for docker_dag to 2021-06-25T19:00:00+00:00
[2021-06-25 16:04:43,687] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.134 seconds
[2021-06-25 16:05:13,725] {scheduler_job.py:182} INFO - Started process (PID=11638) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 16:05:13,725] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 16:05:13,726] {logging_mixin.py:104} INFO - [2021-06-25 16:05:13,726] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 16:05:13,804] {scheduler_job.py:643} INFO - DAG(s) dict_keys(['docker_dag']) retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 16:05:13,833] {logging_mixin.py:104} INFO - [2021-06-25 16:05:13,833] {dag.py:1832} INFO - Sync 1 DAGs
[2021-06-25 16:05:13,850] {logging_mixin.py:104} INFO - [2021-06-25 16:05:13,850] {dag.py:2305} INFO - Setting next_dagrun for docker_dag to 2021-06-25T19:05:00+00:00
[2021-06-25 16:05:13,859] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.137 seconds
[2021-06-25 16:05:43,886] {scheduler_job.py:182} INFO - Started process (PID=11650) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 16:05:43,887] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 16:05:43,887] {logging_mixin.py:104} INFO - [2021-06-25 16:05:43,887] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 16:05:43,970] {scheduler_job.py:643} INFO - DAG(s) dict_keys(['docker_dag']) retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 16:05:43,997] {logging_mixin.py:104} INFO - [2021-06-25 16:05:43,997] {dag.py:1832} INFO - Sync 1 DAGs
[2021-06-25 16:05:44,014] {logging_mixin.py:104} INFO - [2021-06-25 16:05:44,014] {dag.py:2305} INFO - Setting next_dagrun for docker_dag to 2021-06-25T19:05:00+00:00
[2021-06-25 16:05:44,023] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.141 seconds
[2021-06-25 16:06:14,058] {scheduler_job.py:182} INFO - Started process (PID=11668) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 16:06:14,059] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 16:06:14,059] {logging_mixin.py:104} INFO - [2021-06-25 16:06:14,059] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 16:06:14,138] {scheduler_job.py:643} INFO - DAG(s) dict_keys(['docker_dag']) retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 16:06:14,168] {logging_mixin.py:104} INFO - [2021-06-25 16:06:14,168] {dag.py:1832} INFO - Sync 1 DAGs
[2021-06-25 16:06:14,186] {logging_mixin.py:104} INFO - [2021-06-25 16:06:14,186] {dag.py:2305} INFO - Setting next_dagrun for docker_dag to 2021-06-25T19:05:00+00:00
[2021-06-25 16:06:14,198] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.144 seconds
[2021-06-25 16:06:44,217] {scheduler_job.py:182} INFO - Started process (PID=11790) to work on /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 16:06:44,218] {scheduler_job.py:633} INFO - Processing file /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py for tasks to queue
[2021-06-25 16:06:44,219] {logging_mixin.py:104} INFO - [2021-06-25 16:06:44,219] {dagbag.py:487} INFO - Filling up the DagBag from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 16:06:44,309] {scheduler_job.py:643} INFO - DAG(s) dict_keys(['docker_dag']) retrieved from /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py
[2021-06-25 16:06:44,338] {logging_mixin.py:104} INFO - [2021-06-25 16:06:44,337] {dag.py:1832} INFO - Sync 1 DAGs
[2021-06-25 16:06:44,355] {logging_mixin.py:104} INFO - [2021-06-25 16:06:44,355] {dag.py:2305} INFO - Setting next_dagrun for docker_dag to 2021-06-25T19:05:00+00:00
[2021-06-25 16:06:44,370] {scheduler_job.py:190} INFO - Processing /Users/andre/Documents/docker_airflow/dags/dag_docker_teste.py took 0.155 seconds
